rts/Lua/LuaSyncedRead.cpp

```diff
@@ -6453,6 +6453,7 @@ int LuaSyncedRead::GetFactoryCounts(lua_State* L)
 }
 
 
+// FIXME: Consider issuing a deprecation warning and adding @deprecated to either of `GetCommandQueue` or `GetUnitCommands`.
```

Comment:
Do you not already add this to `GetUnitCommands`?

Reply:
Different case, we have repeated `GetCommandQueue` and `GetUnitCommands`, I don't feel comfortable deciding which should be deprecated in this PR, but I definitely agree either of both should (I vote `GetCommandQueue`)

Reply:
I'd also deprecate `GetCommandQueue` and keep `GetUnitCommands` because most unit functions are called GetUnitFoo.